pure-pso/templates/plugin/snapshotclass.yaml

```diff
@@ -0,0 +1,11 @@
+# The snapshotclass will be applied to ose on install
+{{- if eq .Values.orchestrator.name "openshift" }}
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+   name: pure-snapshotclass
+   annotations:
+      snapshot.storage.kubernetes.io/is-default-class: "false"
+driver: pure-csi
+deletionPolicy: Delete
+{{- end }}
```

Comment:
So if it's not an OSE, the yaml file will be empty and applying it does nothing?

Reply:
previous requirement was only installing it on OSE, yea, but the requirement has developed since. if it's k8s, we would like users to decide if they want to apply it, also without the CRD, the install would fail.